cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,28 +1,37 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-
-    cy.get(".ajax_add_to_cart_button > span").click();
-    cy.get(".layer_cart_cart > .button-container").click();
-
-    cy.get(".cart_navigation span").click();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
-    cy.get("#SubmitLogin > span").click();
+import {
+  AddressStepPage,
+  LoginPage,
+  MenuContentPage,
+  PaymentStepPage,
+  ProductsListPage,
+  ShippingStepPage,
+  ShopingCartPage,
+} from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productsListPage = new ProductsListPage();
+const shopingCartPage = new ShopingCartPage();
+const loginPage = new LoginPage();
+const addressStepPage = new AddressStepPage();
+const shippingStepPage = new ShippingStepPage();
+const paymentStepPage = new PaymentStepPage();
 
-    cy.get(".cart_navigation > .button").click();
 
-    cy.get("#cgv").check();
-
-    cy.get(".cart_navigation > .button").click();
-
-    cy.get(".bankwire").click();
-    cy.get("#cart_navigation > .button").click();
-
-    cy.get("#center_column > div > p > strong")
-        .should("have.text", "Your order on My Store is complete.");
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productsListPage.addTShirtToCart();
+    shopingCartPage.goToCartPage();
+    shopingCartPage.goToCheckoutPage();
+    loginPage.login("aperdomobo@gmail.com", "WorkshopProtractor");
+    addressStepPage.goToShippingStep();
+    shippingStepPage.acceptTermsAndConditions();
+    shippingStepPage.goToPaymentStep();
+    paymentStepPage.selectPaymentMethod();
+    paymentStepPage.confirmOrder();
+
+    paymentStepPage.getOrderLabel().should("have.text", "Your order on My Store is complete."); ;
```

Comment:
I see you have the same changes as Salome haha

Reply:
Sorry but could be as a coincidence

Reply:
Its fine, i just think its funny